installscripts/wizard/run.py

```diff
@@ -43,11 +43,9 @@ def check_jenkins_user(url, username, passwd):
 
     if 'authenticated' in open('output').read():
         os.remove('output')
-        os.remove('jenkins-cli.jar')
```

Comment:
Once you rebase with master, please confirm that this is the only change that is needed.